torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -499,6 +500,57 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 # Add additional logic to choose the appropriate memory_config type: DRAM or L1
                 return g.call_function(target_wrappers.clone, args=(args[0],))
 
+            # reference: tt-metal/tests/ttnn/operations/test_group_norm.py::test_group_norm_with_block_sharded_v2_8x8_grid_tile_layout
+            if node.target == torch.ops.aten.native_group_norm.default:
+                arg0_shape = get_shape(args[0])
+                if len(arg0_shape) != 4:
+                    return None
+                N, C, H, W = arg0_shape
+                input_tensor = args[0]
+                weight_tensor = args[1]
+                bias_tensor = args[2]
+                num_groups = args[6]
+                epsilon = args[7]
+                inplace = False
+                grid_size_x = device.compute_with_storage_grid_size().x
+                grid_size_y = device.compute_with_storage_grid_size().y
+                shard_shape = N * H * W // grid_size_x, C // grid_size_y
```

Comment:
The `shaprd_shape` might contains 0 if `N * H * W < grid_size_x` or `C < grid_size_y`. Will that cause problems?

Reply:
Yes this will cause ttnn runtime error, I've add a check to not lowering